src/wasm.h

```diff
@@ -1824,6 +1824,7 @@ class UserSection {
 class DylinkSection {
 public:
   Index memorySize, memoryAlignment, tableSize, tableAlignment;
+  std::vector<char> tail;
```

Comment:
As this is written after the neededDynlibs, how about moving it a line down?

Reply:
> What kind of data appears here? (I've probably missed some recent LLVM work for this.)

Nothing yes, but I'm adding more the moment, and I would like to just work without having to teach binaryen about it (at least not right away).  This allows me to experiment with adding more.

> 
> For a test, maybe roundtripping a binary?

Yup, will add.

Reply:
Where would be good place in binaryen add a binary round-trip test like that?

Reply:
Right now the best we have is `tests/passes/` where you can put a `.wasm` and a `.passes` combination. The passes can include `--roundtrip --print`.

Reply:
I would prefer adding a `test/lit/binary` directory with a `mytest.test` file and `mytest.wasm` file, where `mytest.test` contains RUN lines for performing the test.

Reply:
Added test.  Thanks!